app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ActivitiesController < ApplicationController
+  def create
+    Activity.transaction do
+      params[:activities].each do |activity|
+        @last_activity = Activity.new(activity_params(activity))
```

Comment:
Is there a reason for the instance variable here that gets saved on the next line? Why not just do `Activity.create!(activity_params(activity))`?